mobile/examples/super_resolution/README.md

```diff
@@ -0,0 +1,11 @@
+# ONNX Runtime Mobile Super Resolution Android sample application with Ort-Extensions support for pre/post processing
+
+## Overview
+
+### Model
+
+### Steps to build and run
```

Comment:
We can probably have a general purpose readme in mobile/examples/super_resolution that details creating the model (which could point to the tutorial in ort-ext) and keep this readme specific to the Android example. 

Is the intention for this PR to stay pending until we can plugin pre-release packages, at which point we can flesh out the instructions and check it in? Can update to release packages later when those are available.

Reply:
> We can probably have a general purpose readme in mobile/examples/super_resolution that details creating the model (which could point to the tutorial in ort-ext) and keep this readme specific to the Android example.

ok that makes more sense. was also wondering maybe we should have a general purpose README. will then just include the steps to build/run for android app then.

> Is the intention for this PR to stay pending until we can plugin pre-release packages, at which point we can flesh out the instructions and check it in? Can update to release packages later when those are available.

I guess we can check in the example first, it should be easy to update the aar related instructions. (i.e. no need for creating a local directory and copy aar packages, etc.)

but yeah it also makes more sense if we are not too long before the pre-release version is available. that way, no need for local packages and hosting place for it.